hr_holidays_leave_auto_approve/tests/test_hr_holidays_leave_auto_approve.py

```diff
@@ -2,119 +2,133 @@
 # License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl).
 
 from datetime import datetime, timedelta
+
 from odoo.tests.common import TransactionCase
 
 
 class TestHolidaysAutoValidate(TransactionCase):
     def setUp(self):
         super().setUp()
```

Comment:
@Murtaza-Mithaiwala no need to change indentation or format

Change as per 12, Same as Other

Reply:
When i change code as per your comment and run per-commit in my local code automatically get changed.
can you please tell how can i do ?

Reply:
Then continue with this, No problem